site2/website/data/users.js

```diff
@@ -287,10 +287,10 @@ module.exports = [
         logo_white: true
     },
     {
-        name: 'Kesque',
-        url: 'https://kesque.com',
-        logo: 'https://static.kafkaesque.io/wp-content/uploads/2020/06/kesque_logo.svg',
-        logo_white: true
+        name: 'DataStax',
+        url: 'https://datastax.com',
+        logo: 'https://www.datastax.com/sites/default/files/inline-images/datastax-logotype-negative.png',
+        logo_white: false
```

Comment:
I notice the logo is transparent(white), you need to set it to `true` (adding black background), which would be sth like this:
![image](https://user-images.githubusercontent.com/47805623/107052856-1b7b6c80-6809-11eb-8918-d033980975d9.png)

Or you can set it to "false" or not set it if you use this link for your logo: https://www.datastax.com/_next/static/images/logo-6e2e31572c645818cd3d2ac070d67778.svg

Reply:
You can [build locally](https://github.com/apache/pulsar/tree/master/site2#run-the-site-locally) to check the display effect on Pulsar website.

Reply:
@Jennifer88huang thanks for your suggestion.

This is how the site looks like to me.
I have played with log_white a bit and this looks to me the best option.

![Screenshot 2021-02-05 at 16 26 52](https://user-images.githubusercontent.com/9469110/107053322-f79b3500-67ce-11eb-83b0-c1be061ed08e.png)

I also have the logo with black background and transparent characters, but I was not able to make it work properly
https://www.datastax.com/sites/default/files/inline-images/datastax-logotype-positive.png